NetworkPkg/NvmeOfDxe/NvmeOfCliInterface.c

```diff
@@ -919,6 +931,36 @@ NvmeOfCliProbeCallback (
       CopyMem (Opts->hostnqn, ProbeconnectData->Hostnqn, sizeof (ProbeconnectData->Hostnqn));
     }
   }
+
+  // Fill socket context
+  Private     = (NVMEOF_DRIVER_DATA*)CallbackCtx;
+  AttemptData = &Private->Attempt->Data;
+  Context     = &Private->Attempt->SocketContext;
+
+  Context->Controller = Private->Controller;
+  Context->IsIp6      = AttemptData->SubsysConfigData.NvmeofIpMode == IP_MODE_IP6;
+
+  if (!Context->IsIp6) {
```

Comment:
And what if it _is_ IPv6 ?

Reply:
We don't use Host specific details for creating/connecting to the sockets, while using IPv6. 
https://github.com/timberland-sig/edk2/blob/05081056a30e34019da707fe247731f9f58c0743/NetworkPkg/Library/DxeSpdkLib/SpdkShim/edk_sock.c#L171

Reply:
I think this deserves a comment in the code, too.